backend/codesirius/problems/serializers/sample_test.py

```
@@ -0,0 +1,139 @@
+
+class ValidData(TypedDict):
+    id: NotRequired[int]
+    input: str
+    output: str
+
+
+class SampleTestListSerializer(serializers.ListSerializer):
+
+    @staticmethod
+    def _validate_update(
+        attrs, sample_test_id, existing_sample_tests_map
```

Ruff (0.8.2)

Missing type annotation for function argument `attrs`

(ANN001)